commons/feature_toggles/feature_toggle_manager.py

```diff
@@ -2,19 +2,10 @@
 import json
 from time_tracker_api.security import current_user_email
 from azure.appconfiguration import AzureAppConfigurationClient
-
+from utils.check_variables import  check_variables_are_defined
```

Comment:
Only a little change, it seems we have an extra space in this part `import  check_variables_are_defined`

Reply:
Also, I think we could change the file `check_variables` to something more generic to put other useful functions in there, something like `enviroment_variables`. If we don't have a more generic name for the file, the current name could be stayed.

Reply:
It is a good observation I will correct it